src/Combine.cc

```diff
@@ -489,6 +491,11 @@ void Combine::run(TString hlfFile, const std::string &dataset, double &limit, do
           }
       } 
   }
+  if (floatNuisances_ != "") {
+      RooArgSet toFloat((floatNuisances_=="all")?*nuisances:(w->argSet(floatNuisances_.c_str())));
```

Comment:
a plain `all` may be an issue if some unlucky person has a nuisance called that way and want to float it.
maybe `*` ?